compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -91,6 +92,8 @@ class t_java_generator : public t_oop_generator {
         reuse_objects_ = true;
       } else if( iter->first.compare("option_type") == 0) {
         use_option_type_ = true;
+      } else if( iter->first.compare("handle_runtime_exceptions") == 0) {
```

Comment:
I think the behavior of this java generator option be documented.  Where should it be documented?